collections/_sdk/effects/create_patient_preferred_pharmacies.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "CreatePatientPreferredPharmacies"
+slug: "effect-create-patient-preferred-pharmacies"
+excerpt: "Effect to create preferred pharmacies for a patient."
+hidden: false
+---
+
+## CreatePatientPreferredPharmacies
+
+Creates preferred pharmacies for a patient.
+
+### Parameters
+
+| Name       | Type                             | Description                           |
+|------------|----------------------------------|---------------------------------------|
+| patient_id | `str` or `UUID`                  | The unique identifier of the patient. |
+| pharmacies | `list[PatientPreferredPharmacy]` | List of pharmacies to create.         |
+
+### PatientPreferredPharmacy
+
+The `PatientPreferredPharmacy` dataclass represents a patient's preferred pharmacy, and if it's their default pharmacy.
+
+#### Attributes
+
+| Attribute  | Type   | Description                                          | Required                |
+|------------|--------|------------------------------------------------------|-------------------------|
+| `ncpdp_id` | `str`  | The NCPDC identifier of the pharmacy.                | Yes                     |
```

Comment:
When the effect is interpreted, do we validate the `ncpdp_id` by hitting pharmacy? If not, that's ok, if we do, we should add that context that an invalid id will cause the effect to fail, and link to the pharmacy util for looking a pharmacy up by `ncpdp_id` so a developer can pre-validated.